packages/config/tests/normalize/snapshots/tests.js.md

```diff
@@ -98,6 +116,25 @@ Generated by [AVA](https://ava.li).
       "siteInfo": {}␊
     }`
 
+## Does not assign default functions if build.functions is defined
+
+> Snapshot 1
+
+    `{␊
+      "branch": "branch",␊
+      "buildDir": "/file/path",␊
+      "config": {␊
+        "build": {␊
+          "environment": {},␊
+          "functions": "/file/path"␊
```

Comment:
Should this show `"functions": "test"`?
The only difference I could find in this snapshot is the addition of `"configPath": "/file/path",` which I'm not sure how it is related to handling the functions configuration

Reply:
Yes, it is due to snapshots normalization.
The file paths returned by `@netlify/config` are resolved to absolute paths.
Any absolute path would be different from one machine to another, so we need to normalize them to be able to compare snapshots.
I think this is a more general problem about snapshot testing not fitting our use case here. I have opened https://github.com/netlify/build/issues/2051 to address this.